dronecan_gui_tool/widgets/node_properties.py

```diff
@@ -362,6 +369,7 @@ def __init__(self, parent, node, target_node_id, param_struct, update_callback):
         self._target_node_id = target_node_id
         self._param_struct = param_struct
         self._update_callback = update_callback
+        self._is_tune_editor = param_struct.name == 'TUNE'
```

Comment:
maybe we should rename it to STARTUP_TUNE ? and check it is a string?

Reply:
right, can you rename it in AM32?

Reply:
renamed to STARTUP_TUNE, now also can input empty string to restore to the default startup tune